src/styles/index.scss

```diff
@@ -11,8 +11,6 @@ $font-size-base: 14px;
 
 //PF4 Styles
 $pf-global--enable-reset: false;
-$pf-global--disable-fontawesome: true;
-$pf-global--font-path: '@patternfly/patternfly/assets/fonts/';
 
```

Comment:
PF team told me about add this values...I don't know if we need them now that we migrated 100% to PF4-. Let me check it